### DISCUSSION TITLE
APC mode / Loopy Pro controller mode

### DISCUSSION THREAD

**Discussion by Phonowav** [SynthstromAudible/DelugeFirmware#1391] [Ideas]
APC mode ala APC mini etc. Each pad becomes mappable for clip launching etc, maybe more!

**Comment by m-m-adams** [comment]
It'd be convoluted to set up, but you could shrink a kit keyboard down to 1x1 and make each row a midi command!

**Comment by Phonowav** [comment]
Thanks for the reply Mark! Intriguing about the kit keyboard, could you
elaborate? I’m not sure what you mean. Thanks
